charts/headlamp/values.yaml

```
@@ -104,6 +104,9 @@ serviceAccount:
 clusterRoleBinding:
   # -- Specified whether a cluster role binding should be created
   create: true
+  # -- Cluster role name (If not set: default using "cluster-admin" role)
+  # -- Set name of the Cluster Role with limited permissions from you cluster
+  # clusterRoleName: user-ro
```

```suggestion
  clusterRoleName: cluster-admin
```

It's so that we don't set hardcoded values in chart files, rather than in values.yaml

---

yes, my PR for remove hardcoded values) just set default value like if value not set in values.yaml)
but ok, i'm change code like in you comment)